resources/views/organisations/form/_entry.blade.php

```diff
@@ -42,6 +42,18 @@
     <input type="hidden" name="sync_org_members" value="1">
 @endif
 
+<div class="row">
+    <div class="col-md-6">
+        <div class="form-group">
+            {!! Form::hidden('is_defunct', 0) !!}
+            <label>{!! Form::checkbox('is_defunct', 1, $model->is_defunct )!!}
```

Comment:
I fear creating a new organisation doesn't work anymore ;) Please check

Reply:
Resolved by changing `{!! Form::checkbox('is_defunct', 1, $model->is_defunct)!!}` to `{!! Form::checkbox('is_defunct', 1, $model->is_defunct ?? '' )!!}` page now works properly.